pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -12,16 +12,18 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
 )
 
-type Scanner struct{}
+type Scanner struct{
+	client *http.Client
+}
 
 // Ensure the Scanner satisfies the interface at compile time.
 var _ detectors.Detector = (*Scanner)(nil)
 
 var (
-	client = common.SaneHttpClient()
+	defaultClient = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	domain = regexp.MustCompile(`\b(https:\/\/[a-z0-9-]+\.cloud\.databricks\.com)\b`)
+	domain = regexp.MustCompile(`([a-z0-9-]+(?:\.[a-z0-9-]+)*\.(cloud\.databricks\.com|gcp\.databricks\.com|azurewebsites\.net))`)
```

Comment:
> The domain representation could be without https

@ankushgoel27 Could you provide some documentation on this? I'm hesitant to include this protocol change

Reply:
i am removing the https:// from the regex match and including it in the verifier code. This will help catch any subdomains which are mentioned without the https:// protocol.

Reply:
@zricethezav @4390c336 i have made the changes to the detector. Let me know if anything else needs to be done.